<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QI6C0LV" failures="1" tests="1" name="Default test(failed)" time="0.201" errors="0" timestamp="2023-06-16T16:53:56 IST">
  <testcase classname="testNG.example18_Softassert" name="tc1" time="0.091">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	failed-both are diff expected [hello] but found [hi],
	failed2-result is true expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
failed-both are diff expected [hello] but found [hi],
failed2-result is true expected [false] but found [true]
at testNG.example18_Softassert.tc1(example18_Softassert.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
</testsuite> <!-- Default test(failed) -->
